casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -174,7 +174,7 @@ class RuntimeManager[F[_]: Concurrent: ToAbstractContext] private (
             for {
               _              <- ToAbstractContext[F].fromTask(runtime.space.reset(hash))
               availablePhlos = Cost(deploy.raw.map(_.phloLimit).get)
-              _              <- ToAbstractContext[F].fromTask(runtime.reducer.setAvailablePhlos(availablePhlos))
+              _              <- ToAbstractContext[F].fromTask(runtime.reducer.setPhlo(availablePhlos))
```

Comment:
again, for renames, the call-site changes should come together with the definition-site changes. In this case, the trait extracted in the first commit contains the renames. So either this goes into the first commit, or the first commit is cleaned-up to only contain the trait exxtraction (without method name/signature changes)